photocollage/gtkgui.py

```diff
@@ -20,6 +20,10 @@
 import cairo
 import copy
 import gettext
+import gi
+
+gi.require_version('Gtk', '3.0')
```

Comment:
Sorry, I complety forgot to watch around here! (Without new comments, I don't get any notifications.)